flakey-test-tracking/src/config.js

```diff
@@ -0,0 +1,32 @@
+// shouldTrackFlakes => tests are retried `numRetries` times and flakey results are logged w/ test output
+const shouldTrackFlakes =
+  (process.env.CIRCLECI && process.env.FLAKEY !== 'false') || process.env.FLAKEY === 'true'
+
+// numRetries === times test is run after the initial failure
+const numRetries = process.env.NUM_RETRIES ? Number(process.env.NUM_RETRIES) : 25
```

Comment:
Haha totally. There are a few flakey tests that don't seem to respond to retries so I set it high to try and capture those. I was thinking 10-15 would be more reasonable. What do you think?

Reply:
I think that it doesn't really matter if we catch the flakiness in the first try, the idea is that eventually will happen.
I'm afraid of adding this, and having a lot of PRs which tests fail because are in the beginning of the feature, and having circle taking a long time for them (we have an amount of instances in circle to run everything).
If this just run on master, we could leave a 10. If this could potentially run on every PR, I would start with a small number, and just change it in the future if we see that nothing is catch and it doesn't take too much time.

Reply:
Good thought. I'll change this to be 5 for PRs and 10 when running on master